forester/scripts/compare_performance.py

```
@@ -0,0 +1,322 @@
+#!/usr/bin/env python3
+"""
+Performance Comparison Script for Forester Logs
+Compares queue processing performance between old and new forester versions.
+"""
+
+import re
+import sys
+import argparse
+from datetime import datetime
+from collections import defaultdict
```

Ruff (0.11.9)

`collections.defaultdict` imported but unused

Remove unused import: `collections.defaultdict`

(F401)